src/data/moves/move-attrs/flinch-attr.ts

```diff
@@ -20,7 +20,13 @@ export class FlinchAttr extends AddBattlerTagAttr {
   }
 
   /** Serene Grace and the Water + Fire Pledge combo effect do not stack for flinching */
-  override getMoveChance(user: Pokemon, target: Pokemon, move: Move, showAbility: boolean = false): number {
+  override getMoveChance(
+    user: Pokemon,
+    target: Pokemon,
+    move: Move,
+    _selfEffect: boolean,
```

Comment:
Should there be a `false` default value here ?

```suggestion
    _selfEffect: boolean = false,
```